client/src/pages/certification.tsx

```diff
@@ -17,7 +17,7 @@ function Certification(): JSX.Element {
         path={withPrefix('/certification/:username/:certSlug')}
       />
 
-      <RedirectHome default={true} />
+      <RedirectHome />
```

Comment:
```suggestion
      <RedirectHome default />
```
We have to pass this prop for the component to be rendered. But `default={true}` is a little ugly and this seems nicer :shrug: